utoipa-gen/src/schema_type.rs

```diff
@@ -266,7 +268,10 @@ impl Type<'_> {
 
             #[cfg(feature = "chrono")]
             if !known_format {
-                known_format = matches!(name, "DateTime" | "Date" | "NaiveDate" | "NaiveDateTime");
+                known_format = matches!(
+                    name,
+                    "DateTime" | "Date" | "NaiveDate" | "NaiveTime" | "NaiveDateTime"
```

Comment:
Now I know what is going on here. The issue is that you have defined `NaiveTime` as a known format, but it does not have a format, but only a type which should be string. Now with that known format the naive_time will generate the spec as:
```rust
                .property(
                    "naive_time",
                    utoipa::openapi::ObjectBuilder::new()
                        .schema_type(utoipa::openapi::SchemaType::String)
                        .format(Some()), // <-- note missing format!
                )
```
Note the missing format, it renders empty format because you haven't defined what should be the format of the `NaiveTime`. 
https://github.com/juhaku/utoipa/blob/f8c6d07ef699d4fd6d3eeddfb55d6955d89af0f6/utoipa-gen/src/schema_type.rs#L295-L340

Though in this case I believe the fix would be to remove the `NaiveTime` from  known format function since there is no format specifier for the time like the duration does not have.

Reply:
And then the format in the test should be `null` as in below:
```rust
        "properties.naive_time.format" = r#"null"#, "Post time format"
```